lms/templates/course_modes/track_selection.html

```diff
@@ -16,6 +16,14 @@
 
 <%block name="js_extra">
     <script type="text/javascript">
+        // Edge case: Professional mode uses the checkout page hosted by the E-Commerce service
+        % if use_ecommerce_payment_flow:
+        $('button[name=verified_mode]').click(function(e){
+            e.preventDefault();
+            window.location.href = '${ecommerce_payment_page | n, js_escaped_string}?sku=' +
+            encodeURIComponent('${sku | n, js_escaped_string}');
+        });
+        % endif
```

Comment:
Just for my own awareness, what IS professional mode?  Is this a different kind of course mode, or a term for someone using ecommerce to check out rather than payment MFE?

Reply:
Good question - it's another type of mode, other than `verified` and `audit` we also have `credit`, `professional` and `non-id-professional` which means it's professional mode without ID verification. I'm probably missing another type.. 
And if it's professional mode, it will be using ecommerce to checkout. 

The logic of determining whether it needs to use ecommerce to checkout or not is in `views.py` and it's not changing.

Reply:
The other mode is `honor`!

Reply:
The other course mode type I was missing is `honor` 😁

Reply:
Edit: this is no longer used, the current behavior is that it goes from "enroll" button to Payment MFE (skips track selection). I've removed this from this PR.